app/models/conversion_host.rb

```diff
@@ -202,7 +202,7 @@ def check_conversion_host_role(miq_task_id = nil)
     tag_resource_as('disabled')
   end
 
-  def enable_conversion_host_role(vmware_vddk_package_url = nil, vmware_ssh_private_key = nil, miq_task_id = nil)
+  def enable_conversion_host_role(vmware_vddk_package_url = nil, vmware_ssh_private_key = nil, osp_ca_bundle = nil, miq_task_id = nil)
```

Comment:
Thinking about this now....is there anything that was explicitly passing in a task_id? Otherwise, the param ordering change could be significant.

In the future I think we should change this to keyword params, but that's for another time.

Reply:
I don't think that the order is really important there as nothing else is using it.
But you're right, passing a hash might be a good option, in a follow up.

Reply:
@mturley is it possible the task_id is getting used on a retry or status check?

Reply:
@djberg96 I don't think so, not from the UI anyway. We're only checking status by polling for all tasks, and retry just uses the `request_params` from the existing task to start a new enablement, it doesn't pass the task_id anywhere.